tests/testthat/test-formatNumerics.R

```diff
@@ -0,0 +1,7 @@
+test_that("Formatting numerics work properly", {
+  formatNumerics(log(c(12, 15, 0.3)), digits = 1, scientific = TRUE)
+
+  # Format a data.frame
+  x <- data.frame(parameter = c("a", "b", "c"), value = c(1, 110.4, 6.666))
+  formatNumerics(x, digits = 2, scientific = FALSE)
+})
```

Comment:
If you need or want more detailed testing of the behavior, 
e.g.
```r
expect_equal(
    c("1.00", "110.40", "6.67"),
    formatNumerics(c(1, 110.4, 6.666), digits = 2, scientific = FALSE)
)
expect_equal(
    c("1.00e+00", "1.10e+02", "6.67e+00"),
    formatNumerics(c(1, 110.4, 6.666), digits = 2, scientific = TRUE)
)
```

Reply:
Did you my comment here: https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/issues/8#issuecomment-979066820?

I don't think the function is working as expected.